x-pack/legacy/plugins/siem/public/components/page/hosts/authentications_over_time/translation.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const AUTHENTICATIONS_COUNT = i18n.translate(
+  'xpack.siem.authenticationsOverTime.authenticationCount',
```

Comment:
Could be xpack.siem.authenticationsOverTime.authenticationCountTitle. 
There's a specific naming rules we can follow here:
https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#message-types